splunk-quarkus/src/main/java/com/redhat/console/integrations/ErrorHandlingRoutes.java

```diff
@@ -38,73 +44,57 @@ public void configure() throws Exception {
         configureSecureConnectionFailed();
     }
 
-    private void configureSecureConnectionFailed() throws Exception {
-        Processor ceEncoder = new CloudEventEncoder(COMPONENT_NAME, RETURN_TYPE);
-        Processor resultTransformer = new ResultTransformer();
+    private void configureSecureConnectionFailed() {
         // The error handler when we receive an HTTP (unsecure) connection instead of HTTPS
         from(direct("secureConnectionFailed"))
                 .routeId("secureConnectionFailed")
                 .log(LoggingLevel.ERROR, "ProtocolException for event ${header.ce-id} (orgId ${header.orgId}"
-                                         + " account ${header.accountId}) to ${header.targetUrl}: ${exception.message}")
+                                         + " account ${header.accountId}) to ${exchangeProperty.targetUrl}: ${exception.message}")
                 .log(LoggingLevel.DEBUG, "${exception.stacktrace}")
-                .setBody(simple("${exception.message}"))
-                .setHeader("outcome-fail", simple("true"))
```

Comment:
There's a better way than headers to pass data within a route or between two routes: exchange properties. Headers are eventually sent to the remote server if an HTTP call is performed while exchange properties are not.